README.md

```diff
@@ -34,6 +34,19 @@ Furthermore, we welcome contributions regarding...
 + computational performance
 + model insight
 
+### How to run the code and notebooks
+To be able to contribute, you'll need to be able to run the code and notebooks on this repo. To that end, we strongly recommend [installing the  Anaconda distribution](https://www.anaconda.com/products/individual) for python. This will make sure all the packages and necessary compilers come from the same source, and it will allow you to create dedicated virtual environments to run your code safely for each project independently.
+
+Once Anaconda is installed, use the terminal and make sure you're in the root of the rtlive-global repository. Then, follow the following steps (still in the terminal):
```

Comment:
Since the environment is created with `conda create`, should we really recommend Anaconda? (It's sooo biig!) In my understanding you should get the same result with [miniconda](https://docs.conda.io/en/latest/miniconda.html).

Reply:
Good point, just pushed the change 😉

Reply:
Your edit was incomplete.. Read the whole thing. Sorry dude. You can write Python with a capital P.

Reply:
Should be all good now